app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,127 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light-gray: #777;
+$orange: #F25204;
+$dark-blue: #093D5E;
+$brown: #5C2D22;
+$light-brown: #9E4D0B;
+$tan: #E7CE9F;
+$purple: #800080;
+
+// From adobe https://color.adobe.com/Theme-20-color-theme-5544017/
+$grey: #D9C7C1;
+$yellow: #F2CA50;
+$orange_yellow: #F2994B;
+$adobe_orage: #F27B35;
+$green: #344021;
+
+// https://color.adobe.com/Old-Orange-Peel-color-theme-448736/
+$dark-brown1: #382500;
+$red1: #992200;
+
+// https://color.adobe.com/Infographics-6-color-theme-5378443/
+$dark-blue2: #314655;
+$orange2: #F58625;
+$red2: #F05026;
+$cyan2: #31BDC1;
+
+html {
+  background-color: $dark-blue2;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  padding-top: 150px;
+  margin: 0 -35px;
+  color: $orange2;
+  align-top: center;
+  background-color:rgba(255,255,255,0) !important;
+}
+
+.main-body {
+  padding-top: 10px;
+  text-align: center;
+  // // background: blue;
+  // width: 110%;
+  // padding-right: -15px;
+}
+
+.main-menu {
+  // background-color:rgba(255,255,255,.7);
+  padding: 30px;
+}
+
+// .main-item {
+//   padding: 15px;
+//   font-size: 2em;
+//   letter-spacing: 2px;
+// }
+
+li {
+  list-style-type: none;
+  padding: 3px 30px 20px 0;
+}
+
+.navbar {
+  background: $brown;
+  font-size: 1.6em;
+}
+
+a.nav-list-item {
+  color: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Properties should be ordered background-color, color, letter-spacing, text-decoration